eth/backend.go

```diff
@@ -490,6 +505,18 @@ func (s *Ethereum) setupDiscovery() error {
 		s.discmix.AddSource(iter)
 	}
 
+	// Add DHT nodes from discv4.
+	if s.p2pServer.DiscoveryV4() != nil {
+		iter := s.p2pServer.DiscoveryV4().RandomNodes()
+		resolverFunc := func(ctx context.Context, enr *enode.Node) (*enode.Node, error) {
```

Comment:
I don't like the error return. The error is discarded anyway, so why return it here?

Reply:
Good point, fixing.